skrf/plotting.py

```diff
@@ -1448,6 +1448,80 @@ def plot_minmax_bounds_component(self: NetworkSet, attribute: str, m: int = 0, n
     scale_frequency_ticks(ax, ntwk_mean.frequency.unit)
     ax.axis('tight')
 
+@axes_kwarg
+def plot_violin_component(
+        self: NetworkSet,
+        attribute: str,
+        m: int = 0,
+        n: int = 0,
```

Comment:
```suggestion
        n: int = 0,
        *, 
```

The single star forces the users to pass all the following arguments keyword only, which is probably more self explaing.

Reply:
I like this suggestion, going to apply it to the similar uncertainty bounds and minmax bounds methods as well. Also helped find a bug in all these methods where they couldn't accept any positional arguments because they would get passed into the "attribute" argument.